client/e2e/pages/authenticated/dashboard/profile/page.spec.ts

```diff
@@ -0,0 +1,81 @@
+// 🧪 Suite to test the onboarding\Home page `http://localhost:3000/home`
+
+import { test } from "@playwright/test";
+// 🪄 Page Object Models
+import { DashboardPOM } from "@/e2e/poms/dashboard";
+import { ProfilePOM } from "@/e2e/poms/profile";
+// ☰ Enums
+import { UserRole } from "@/e2e/utils/enums";
+// 🥞 Connection pool to postgres DB
+import { pool } from "@/e2e/utils/pool";
+import { deleteNewUser, upsertIOUser } from "@/e2e/utils/queries";
+// ℹ️ Environment variables
+import * as dotenv from "dotenv";
+dotenv.config({ path: "./e2e/.env.local" });
+
+// 🛠️ Function: deletes the new user record from the database
+const deleteNewUserRecord = async () => {
+  try {
+    const query = deleteNewUser;
+    // Execute the deletion query
+    await pool.query(query);
+  } catch (error) {
+    // eslint-disable-next-line no-console
+    console.error("Error deleting new user record:", error);
+    throw error;
+  }
+};
+// 📚 Declare a beforeAll hook that is executed once per worker process before all tests.
+// 🥞 Set DB for profile update
+// For industry_user, ensure there is an associated user
+// For no role/new user, ensure there is NOT an associated user
+test.beforeAll(async () => {
+  try {
+    // 👤 industry_user: bc-cas-dev-secondary
+    // Upsert a User record
+    let query = upsertIOUser;
+    // ▶️ Execute the query
+    await pool.query(query);
+
+    // 👤 new user: bc-cas-dev-three
+    // Delete User record
+    await deleteNewUserRecord();
+  } catch (error) {
+    // eslint-disable-next-line no-console
+    console.error("❌ Error in Db setup for profile roles:", error);
+    throw error;
+  }
+});
+
+// 🏷 Annotate test suite as serial
+test.describe.configure({ mode: "serial" });
+test.describe("Test Page - Profile", () => {
+  // ➰ Loop through the entries of UserRole enum
+  for (let [role, value] of Object.entries(UserRole)) {
+    test.describe(`Test User Role - ${value}`, () => {
+      // 👤 run test as this role
+      role = "E2E_" + role;
+      const storageState = process.env[role + "_STORAGE"] as string;
+      test.use({ storageState: storageState });
+
+      test("Test Profile Update", async ({ page }) => {
+        // 🛸 Navigate to profile page
+        const profilePage = new ProfilePOM(page);
+        await profilePage.route();
+        // 🔍 Assert profile update required fields
+        await profilePage.updateFail();
+        // 🔍 Assert profile update
+        await profilePage.updateSuccess();
```

Comment:
`updateSuccess` fills out the form, but I think we're missing an assertion to check if the form actually submitted correctly. In most of the test I think it's going to be a redirect to a confirmation page, but for profile, I think we expect to see the little checkmark on the submission button on success

Reply:
I swear, someone stole me code! reverted little checkmark check